src/client/components/Header.tsx

```diff
@@ -39,10 +61,43 @@ export const Header: React.FunctionComponent = (props) => {
             ? null :
               isLoggedIn
                 ? [
-                    <IconButton color="inherit" key={0}><SettingsIcon color="inherit"/></IconButton>,
+                    <IconButton color="inherit" key={0} onClick={() => setOpen(true)}><SettingsIcon color="inherit" /></IconButton>,
                     <Fragment key={1}>{data.me.name}</Fragment>,
-                    <Button onClick={logOutUser} key={2}>Log Out</Button>
+                    <Button onClick={logOutUser} key={2}>Log Out</Button>,
+                    <Modal open={open} style={{
+                      display: 'flex',
+                      alignItems: 'center',
+                      justifyContent: 'center',
+                    }}>
+                      <Paper style={{
```

Comment:
Try adding some padding? like 10px so it isn't so close to the top and bottom of the modal